src/popup/router/components/dashboard/Card.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div
+    class="card"
+    :class="{ 'is-big-card': isBigCard, hover }"
+    :style="styleComponent"
+    @click="(e) => $emit('click', e)"
+  >
+    <div class="card-icon">
+      <slot
+        name="icon"
+        class="icon-svg"
+      />
+    </div>
+    <div>
+      <div class="title">
+        {{ title }}
+      </div>
+      <div class="description">
+        {{ description }}
+      </div>
+      <slot />
+    </div>
+  </div>
+</template>
+
+<script>
+
+export default {
+  name: 'Card',
+  props: {
+    title: {
+      type: String,
+      default: '',
+      required: true,
+    },
+    description: {
+      type: String,
+      default: '',
+      required: true,
+    },
+    background: {
+      type: String,
+      default: null,
+    },
+    isBigCard: Boolean,
+    hover: Boolean,
+  },
+  computed: {
+    styleComponent() {
+      return {
+        backgroundImage: (this.background)
+          ? `url("${this.background}")`
+          : null,
+      };
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@use '../../../../styles/variables';
+@use '../../../../styles/typography';
+
+.card {
+  width: 100%;
+  background-color: #262626;
+  min-height: 116px;
+  border-radius: 10px;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  gap: 8px;
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  &.is-big-card {
+    flex-direction: row;
+    align-items: start;
+    gap: 16px;
+    text-align: left;
+  }
+
+  &.hover {
+    cursor: pointer;
+    transition: 0.4s;
+
+    &:hover {
+      filter: brightness(1.4);
```

Comment:
I think this is not the effect Tsvetan wanted to achieve. Please align with him or check the designs. You can use the "Play" icon on the top of figma to preview to preview the interactions.